intro-Cloud/ch_Flavors.tex

```diff
@@ -62,7 +62,18 @@ \section{Instance Types}\label{sec:instance-types}
 \section{Flavor Sizes}\label{sec:flavor-sizes}
 A flavor size is a set of virtualized hardware resouces to a virtual
 machine (VM) instance like system memory size, virtual CPU (vCPU)
-or the root filesystem size.
+or the root filesystem size. 
+
+The flavor's root size is the amount of disk space
+used by the root (\emph{/}) partition, an ephemeral disk that the base image is copied into
+(see section \ref{launch-an-instance} for more information about VM persistent/non-persistent
+instances).
+The flavor's root storage is only taken into account when booting from a non-persistent VM but not from a
```

Comment:
what exactly is meant here by 'taken into account'?

Reply:
on creation of a non-persistent VM you get this disk size, on creation of a persistent VM you can pick whatever size you want?

Reply:
@wpoely86 ok I have to clarify this a bit more.. The root size value is only used when you instantiate the VM the first time, but you can choose to  make it persistent creating a volume with the flavor's root size (which you can resize before), or not, in that case is a non-persistent VM.